services/121-service/package.json

```diff
@@ -82,6 +82,7 @@
     "csv-parser": "^3.2.0",
     "easy-soap-request": "^5.7.0",
     "form-data": "^4.0.4",
+    "he": "^1.2.0",
```

Comment:
This seems nice-and-focussed for our purpose..
Looking through the (open) issues on GitHub for this, there might be some room-for-improvement that another package might fit our use-case better (or more specifically, or 'better when using with typescript'...

But lets see how this works out for now...